frontend/pages/DashboardPage/cards/ActivityFeed/components/ScriptDetailsModal/ScriptDetailsModal.tsx

```diff
@@ -87,9 +87,31 @@ const StatusMessage = ({
     case -2:
       // Expected API message: "Scripts are disabled for this host. To run scripts, deploy the fleetd agent with scripts enabled."
       return <StatusMessageError message={message} />;
-    case -1:
-      // Expected API message: "Timeout. Fleet stopped the script after 5 minutes to protect host performance."
-      return <StatusMessageError message={message} />;
+    case -1: {
+      // message should look like: "Timeout. Fleet stopped the script after 600 seconds to protect host performance.";
+      const timeOutValue = message.match(/(\d+\s(?:seconds))/);
+
+      let varText = null;
```

Comment:
Is this cleaner?

`const varText = timeOutValue ?  <>
            after{" "}
            <TooltipWrapper tipContent="Timeout can be configured by updating agent options.">
              {timeOutValue[0]}
            </TooltipWrapper>{" "}
          </> : null;`

Reply:
Yea, good call